x-pack/plugins/rule_registry/server/alert_data_client/alerts_client_factory.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ElasticsearchClient, KibanaRequest, Logger } from 'src/core/server';
+import { PublicMethodsOf } from '@kbn/utility-types';
+import { SecurityPluginSetup } from '../../../security/server';
```

Comment:
Doesn't this create a dependency on the security plugin? If so it needs to be specified in kibana.json and tsconfig.json.